public/scripts/editor/js/fc/filesystem-sync.js

```diff
@@ -52,11 +54,11 @@ define(["jquery"], function($) {
         return;
       }
 
-      options.data = JSON.stringify({
-        path: path,
-        buffer: data,
-        dateUpdated: (new Date()).toISOString()
-      });
+      var formData = options.data = new FormData();
+      formData.append("dataUpdated", (new Date()).toISOString());
+      // Don't worry about actual mime type, just treat as binary
+      var blob = new Blob([data], {type: "application/octet-stream"});
+      formData.append("brambleFile", blob, path);
```

Comment:
I wonder if sending the path like this is such a good idea because of [this](https://developer.mozilla.org/en-US/docs/Web/API/FormData/append#Browser_compatibility). It says that append with filename may or may not work with IE, Opera, Safari, etc. It might be safer to just send it as a form field?